src/Sentry.Bindings.Android/Sentry.Bindings.Android.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>net7.0-android</TargetFramework>
     <!-- BG8605 and BG8606 happen because there's a missing androidx.lifecycle dependency, but we don't need it here.  (The native Android Sentry SDK will use it if it exists.) -->
     <NoWarn>$(NoWarn);BG8605;BG8606</NoWarn>
-    <SentryAndroidSdkVersion>7.16.0</SentryAndroidSdkVersion>
+    <SentryAndroidSdkVersion>7.15.0-alpha.1</SentryAndroidSdkVersion>
```

Comment:
using `7.15.0-alpha.1` to change the NDK handler strategy and be able to get the logcat file attached see #3694.

To be changed before emerging to `main`